docs/releasing/publishing.md

```diff
@@ -77,23 +77,25 @@ Developers should pair on releases. When remote working, it can be useful to be
 
 11. Once a reviewer approves the pull request, merge it to **main**.
 
-12. Checkout the **main** branch and pull the latest changes.
+## Publish a release to npm
 
-13. Log into npm (`npm login`), using the npm/govuk-patterns-and-tools team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/main/npm/govuk-patterns-and-tools).
+1. Check out the **main** branch and pull the latest changes.
 
-14. Run `npm run publish-release`. You will now be prompted to continue or cancel.
+2. Log into npm (`npm login`), using the npm/govuk-patterns-and-tools team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/main/npm/govuk-patterns-and-tools).
```

Comment:
Can we update https://github.com/alphagov/govuk-frontend/blob/main/docs/releasing/publishing-from-a-support-branch.md to match?

Reply:
Definitely! I'm also going to change 'log into' to 'sign in to', based on [GOV.UK style guide advice](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#sign-in-or-log-in):

> Use sign in rather than log in (verb) for calls-to-action where users enter their details to access a service.

```suggestion
2. Sign in to npm (`npm login`), using the npm/govuk-patterns-and-tools team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/main/npm/govuk-patterns-and-tools).
```